.mypy.ini

```
@@ -34,3 +34,6 @@ strict_optional = true
 warn_no_return = true
 warn_redundant_casts = true
 warn_unused_ignores = true
+
+[mypy-awx.main.models.credential.*] # This can be removed once AWX has been imported
```

It's customary to have double whitespaces before comments in almost any language, so it's best to keep that habit even where it's not linted, for consistency.
```suggestion
[mypy-awx.main.models.credential.*]  # This can be removed once AWX has been imported
```